UPDATING.md

```diff
@@ -61,6 +61,29 @@ https://developers.google.com/style/inclusive-documentation
 
 -->
 
+### Custom executors is loaded using full import path
+
+In previous versions of Airflow it was possible to use plugins to load custom executors. It is still
+possible, but the configuration has changed. Now you don't have to create a plugin to configure a
+custom executor, but you need to provide the full path to the module in the `executor` option
+in the `core` section. The purpose of this change is to simplify the plugin mechanism and make
+it easier to configure executor.
+
+If your module was in the path `my_acme_company.executors.MyCustomExecutor`  and the plugin was
+called `my_plugin` then your configuration looks like this
+
+```ini
+[core]
+executor = my_plguin.MyCustomExecutor
```

Comment:
Did this even work before? I thought we only looked at a specific hard-coded list?

Reply:
More importantly for updating -- did we ever publish executors-via- plugin to 1.10 release, if not then these updating instructions don't make sense.

Reply:
II haven't tested it, but the code looked like it had a chance to work.  In the documentation, we have information:
https://airflow.readthedocs.io/en/1.10.6/plugins.html

Reply:
Oh wow missed that one totally!